.github/workflows/notebook-test.yml

```diff
@@ -31,6 +31,10 @@ jobs:
       - name: Get all changed files
         id: all-changed-files
         uses: tj-actions/changed-files@af2816c65436325c50621100d67f6e853cd1b0f1
+        with:
+          files: docs/**/*.ipynb
+          files_ignore: docs/api/**
```

Comment:
Not actually necessary. API docs never have notebooks. And if we had them in `api/migration_guides`, those should be checked.

```suggestion
```